lld/ELF/InputFiles.cpp

```diff
@@ -1797,18 +1799,18 @@ void BitcodeFile::parse() {
 void BitcodeFile::parseLazy() {
   numSymbols = obj->symbols().size();
   symbols = std::make_unique<Symbol *[]>(numSymbols);
-  for (auto [i, irSym] : llvm::enumerate(obj->symbols()))
+  for (auto [i, irSym] : llvm::enumerate(obj->symbols())) {
+    // Keep copies of per-module undefined symbols for LTO::GlobalResolutions
```

Comment:
Maybe update the comment a bit since it only talks about undefined symbols but now applies to both. Probably pull up the other comment below on line 1806 and combine them?

Reply:
done.